chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/AnnouncementManager.java

```diff
@@ -2,33 +2,109 @@
 
 import java.util.List;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.support.ApplicationObjectSupport;
 import org.springframework.security.access.AccessDeniedException;
+import org.springframework.security.access.prepost.PostFilter;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
 
 import com.fasterxml.jackson.core.JsonProcessingException;
 
-import gov.healthit.chpl.dto.AnnouncementDTO;
+import gov.healthit.chpl.dao.AnnouncementDAO;
+import gov.healthit.chpl.domain.Announcement;
+import gov.healthit.chpl.domain.activity.ActivityConcept;
 import gov.healthit.chpl.exception.EntityCreationException;
 import gov.healthit.chpl.exception.EntityRetrievalException;
 import gov.healthit.chpl.exception.UserRetrievalException;
 
-public interface AnnouncementManager {
+@Service
+public class AnnouncementManager extends ApplicationObjectSupport {
 
-    AnnouncementDTO create(AnnouncementDTO announcement)
-            throws UserRetrievalException, EntityCreationException, EntityRetrievalException, JsonProcessingException;
+    @Autowired
+    private AnnouncementDAO announcementDAO;
 
-    AnnouncementDTO update(AnnouncementDTO announcement)
-            throws EntityRetrievalException, JsonProcessingException, EntityCreationException;
+    @Autowired
+    private ActivityManager activityManager;
```

Comment:
Before we go to STG - these @Autowired variable should be moved to a constructor.